config/locales/en.yml

```diff
@@ -78,6 +78,18 @@ en:
     claims_preventing_payment_message:
       one: This claim cannot currently be approved because we’re already paying another claim (%{claim_references}) to this claimant in this payroll month using different payment details. Please speak to a Grade 7.
       other: This claim cannot currently be approved because we’re already paying other claims (%{claim_references}) to this claimant in this payroll month using different payment details. Please speak to a Grade 7.
+    payment_deletion_claim_warning:
+      one: This means claim %{references} has not been included in this pay run.
+      other: This means claims %{references} have not been included in this pay run.
+    payment_deleted_title:
+      one: You have removed a claim from the payroll confirmation report
+      other: You have removed %{count} claims from the payroll confirmation report
+    payment_deleted_outcome_message:
+      one: Claim %{references} has not been paid in this pay run.
```

Comment:
```suggestion
      one: Claim %{references} will not be paid in this pay run.
```

Reply:
I get where you're coming from here, the claim not being paid is not a side effect of us deleting the payment. This has already been decided by Cantium